pkgs/by-name/sp/spacedrive/package.nix

```diff
@@ -63,15 +63,16 @@ else appimageTools.wrapType2 {
   extraInstallCommands =
     let
       appimageContents = appimageTools.extractType2 { inherit pname version src; };
+      desktopFile = "com.${pname}.desktop";
     in
     ''
       # Remove version from entrypoint
-      mv $out/bin/spacedrive-"${version}" $out/bin/spacedrive
+      mv $out/bin/${pname}-${version} $out/bin/${pname}
 
       # Install .desktop files
-      install -Dm444 ${appimageContents}/spacedrive.desktop -t $out/share/applications
-      install -Dm444 ${appimageContents}/spacedrive.png -t $out/share/pixmaps
-      substituteInPlace $out/share/applications/spacedrive.desktop \
-        --replace 'Exec=AppRun --no-sandbox %U' 'Exec=spacedrive'
+      install -Dm444 ${appimageContents}/${desktopFile} -t $out/share/applications
+      install -Dm444 ${appimageContents}/${pname}.png -t $out/share/pixmaps
+      substituteInPlace $out/share/applications/${desktopFile} \
+        --replace 'Exec=usr/bin/${pname}' 'Exec=${pname}'
```

Comment:
I'm not sure these changes should be applied. See, https://github.com/NixOS/nixpkgs/pull/270121#discussion_r1405432291

Reply:
@MikaelFangel would you prefer having raw naming in those fields ?